QUEST_LV_0100.tsv

```diff
@@ -4198,176 +4198,176 @@ QUEST_LV_0100_20150317_004197	$Destroy the West Barrier in Nuotori Cliff
-QUEST_LV_0100_20150317_004302	{memo X}You found a part at the Entracne Purifier. Assemble it to the Purifier.
+QUEST_LV_0100_20150317_004283	$Collect Golden Pieces
+QUEST_LV_0100_20150317_004284	$Go back to Davio to complete the medicine
+QUEST_LV_0100_20150317_004285	$Go back to Jonas and give him the records. Make sure he takes the medicine that Davio gave to you.
+QUEST_LV_0100_20150317_004286	$Go back to Jonas and give him the records. Make sure he takes the medicine that Davio gave to you. Jonas is probably at the Abandoned Old Workshop.
+QUEST_LV_0100_20150317_004287	$Talk to Mercenary Mirta
+QUEST_LV_0100_20150317_004288	$Mirta seems to be severely injured. Ask him why he is injured and where are the supplies to be sent to Kepeck.
+QUEST_LV_0100_20150317_004289	$Retrieve the supplies to be sent to Kepeck
+QUEST_LV_0100_20150317_004290	$Mercenary Mirta says that he lost the supplies from monsters' attacks. Retrieve the supplies and send them to Kepeck on behalf of Mirta.
+QUEST_LV_0100_20150317_004291	$Find Historian Kepeck at Apvija Hills
+QUEST_LV_0100_20150317_004292	$It seems that you retrieved most of the supplies. Give them to the Historian Kepeck at Apvija Hills.
+QUEST_LV_0100_20150317_004293	$Talk with Vaidotas in the Mine
+QUEST_LV_0100_20150317_004294	$Enter the Crystal Mine and talk to Vaidotas.
+QUEST_LV_0100_20150317_004295	$Repair the Purifiers on the 1st Floor
+QUEST_LV_0100_20150317_004296	$Repair all the Purifiers on the 1st floor. Some are easy to repair, while others that are more difficult to repair will need the use of the compass Vaidotas gave you.
+QUEST_LV_0100_20150317_004297	$Inspect the Entrance Purifier on the 1st Floor
+QUEST_LV_0100_20150317_004298	$The Purifiers in the Crystal Mine are not working and toxic fumes are present. Activate the Purifiers to clean the air.
+QUEST_LV_0100_20150317_004299	{memo X}Search for Purifier Parts on the floor
+QUEST_LV_0100_20150317_004300	{memo X}Check the floor. You need to find missing parts for the Purifier.
+QUEST_LV_0100_20150317_004301	{memo X}Assemble the Entrance Purifier on the 1st Floor
+QUEST_LV_0100_20150317_004302	{memo X}You found a part for the Entrance Purifier. Assemble it to the Purifier.
 QUEST_LV_0100_20150317_004303	{memo X} Find %s on the floor
-QUEST_LV_0100_20150317_004304	Retrieve Purifier parts
-QUEST_LV_0100_20150317_004305	Retrieve purifier parts from the Vubbe with question mark on its head.
-QUEST_LV_0100_20150317_004306	Repair Entrance Purifier on the 1st Floor
-QUEST_LV_0100_20150317_004307	Found the missing parts of the Entrance Purifier. Assemble and activate the purifier with it.
+QUEST_LV_0100_20150317_004304	$Retrieve Purifier Parts
+QUEST_LV_0100_20150317_004305	$Retrieve Purifier Parts from the Vubbe with a question mark on its head.
+QUEST_LV_0100_20150317_004306	$Repair the Entrance Purifier on the 1st Floor
+QUEST_LV_0100_20150317_004307	$Found the missing parts of the Entrance Purifier. Try assembling and activating the purifier.
```

Comment:
Try assembling and activating the purifier. -> Try to assemble and activate it (*shorter*)